rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -253,19 +253,31 @@ public RMState getState(@HeaderParam("sessionid") String sessionId) throws NotCo
     }
 
     /**
-     * Returns the initial state of the resource manager
+     * Returns difference between current state of the resource manager
+     * and state that the client is aware of. Each event that changes state fo the RM
+     * has counter assosiacted to it. Thus client has to provide 'latestCounter',
+     * i.e. latest event he is aware of.
      * @param sessionId a valid session id
-     * @return the initial state of the resource manager 
+     * @param clientCounter (optional) is the latest counter client has, if parameter is not provided then
+     *                                 method returns all events
+     * @return the difference between current state and state that client knows
```

Comment:
Will you adapt to this change all the users of this end point?

Reply:
1) I thought there is no other users except rm portal
2) this change extends current contract (if you send request where latest counter 100 then you will not get first 100 events, if you don't send anything then you get response as before)